src/index.html

```diff
@@ -22,6 +22,286 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="moyo logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__item">
+          <li class="nav__link">
+            <a class="nav__a is-active">Apple</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__a">Samsung</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__a">Smartphones</a>
+          </li>
+          <li class="nav__link">
+            <a
+              class="nav__a"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__a">Gadgets</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__a">Tablets</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__a">Photo</a>
+          </li>
+          <li class="nav__link">
+            <a class="nav__a">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE I-Mac"
+          class="card__photo"
+        />
+        <h2 class="card__productname">
+          APPLE A1419 iMac 27"&nbsp;Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h3 class="card__productcode">Product code: 195434</h3>
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <p class="stars__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-word">Price:</p>
+          <p class="card__price-number">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__buy-button"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```